.husky/pre-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+while read local_ref local_sha remote_ref remote_sha; do
+  # Check if the local branch name starts with "docs-release-pr", in which case exit successfully
+  if [[ "$local_ref" =~ ^refs/heads/docs-release-pr-.* ]]; then
```

Comment:
We should add `changest-release` too, because `changeset` will try to bump `CHANGELOG`, `package.json` when it creates "Version Packages" PR.

Reply:
Good catch, I think we should be good with the change to only target `src` folder though.

Reply:
Just checked “Version Packages” PR, it changes src/version.ts so I think we need to add changeset-release/main to the ignorelist 👍

Reply:
Yeah, it will bump `react/src/version.ts`. 🥴

Reply:
Thanks for checking, added!